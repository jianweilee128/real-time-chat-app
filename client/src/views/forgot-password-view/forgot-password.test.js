import React from "react";
import { shallow } from "enzyme";
import ForgotPasswordView from "./forgot-password-view";

describe("ForgotPasswordView", () => {
  let wrapper;
  let mockForgotPassword;
  beforeEach(() => {
    mockForgotPassword = jest.fn();

    const mockProps = {
      forgotPasswordSuccess: false,
      forgotPassword: mockForgotPassword,
    };

    wrapper = shallow(<ForgotPasswordView.WrappedComponent {...mockProps} />);
  });
  it("should render ForgotPasswordView component", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
