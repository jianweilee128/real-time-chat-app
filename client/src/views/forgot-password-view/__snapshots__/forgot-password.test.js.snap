// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ForgotPasswordView should render ForgotPasswordView component 1`] = `ShallowWrapper {}`;
